plottr/apps/inspectr.py

```diff
@@ -459,7 +488,12 @@ def setDateSelection(self, dates: Sequence[str]) -> None:
         if len(dates) > 0:
             assert self.dbdf is not None
             selection = self.dbdf.loc[self.dbdf['started_date'].isin(dates)].sort_index(ascending=False)
-            self.runList.setRuns(selection.to_dict(orient='index'))
+            old_selection = [item.text()
+                             for item in self.dateList.selectedItems()]
+            if dates not in old_selection:
```

Comment:
This logic is not correct. `dates` is a list of strings and so is `old_selection` so this will always evaluate to false. I think what you want to do is `if not all(data in old_selection for data in dates):`